samples/Friends/Friends/Program.cs

```diff
@@ -15,8 +15,9 @@ namespace Friends
 {
     class Friends
     {
-        public unsafe static void Main(string[] args)
+        public static unsafe void Main(string[] args)
         {
+            TrinityConfig.CurrentRunningMode = RunningMode.Embedded;
```

Comment:
Please revert.

Reply:
Just altered code formatting - that line of code has always been there